src/components/LayoutContainer/index.tsx

```diff
@@ -0,0 +1,8 @@
+import { LayoutContainerProps } from 'components/LayoutContainer/types';
```

Comment:
❗️
Здесь нет причин ударяться в алиасы
```suggestion
import { LayoutContainerProps } from './types';
```

Reply:
согласен